BridgeCareApp/VuejsApp/src/components/investment-editor/InvestmentEditor.vue

```diff
@@ -426,7 +406,7 @@
          * Clears the selected investment library by setting selectItemValue to an empty value or 0
          */
         onClearSelectedInvestmentLibrary() {
-            this.selectItemValue = hasValue(this.selectItemValue) ? '' : '0';
+            this.selectItemValue = hasValue(this.selectItemValue) ? '' : '';
```

Comment:
This change makes the ternary redundant.

Reply:
So, I will just add `this.selectItemValue = '';`
Is it okay?

Reply:
Were you using `0` for some special scenario, which you think can break. I have tested a lot, and thinks are working fine. But I might be missing something